vulture_os/toolkit/api_parser/cybereason/malop.py

```diff
@@ -0,0 +1,377 @@
+
+    @staticmethod
+    def _process_details_flattener(process_details: dict) -> list:
+        flattened_process_details = [
+            {
+                'name': v['simpleValues'].get('imageFile.name', {}).get('values', []),
+                'cmdline': v['simpleValues'].get('commandLine', {}).get('values', []),
+                'hash': {
+                    'md5': v['simpleValues'].get('imageFile.md5String', {}).get('values', []),
+                    'sha1': v['simpleValues'].get('imageFile.sha1String', {}).get('values', []),
+                    'sha256': v['simpleValues'].get('imageFile.sha256String', {}).get('values', []),
+                },
+                'signature': {
+                    'product_name': v['simpleValues'].get('imageFile.productName', {}).get('values', []),
+                    'company_name': v['simpleValues'].get('imageFile.companyName', {}).get('values', []),
+                },
+                'parent': {
+                    'name': v['simpleValues'].get('parentProcess.name', {}).get('values', []),
+                    'machine': v['simpleValues'].get('ownerMachine.name', {}).get('values', []),
+                },
+            } for v in process_details.values()
+        ]
+        return flattened_process_details
+
+    @staticmethod
+    def _malops_details_flattener(malop_details: dict) -> list:
+        flattened_process_details = [
+            {
+                "name": i.get('elementDisplayName', ''),
+                "path": i.get('correctedPath', ''),
+                "hash": {"sha1": i.get('sha1String', '')},
+                "machine": {"hostname": i.get('ownerMachineName', '')},
+                "action": i.get('detectionDecisionStatus', ''),
+                "last_seen": i.get('lastSeen', ''),
+            } for i in malop_details.get('fileSuspects', []) or []
+        ]
+        return flattened_process_details
+
+    def _get_sensors(self, guids: list) -> list:
+        sensors_url = f"{self.instance.host}/{self.SENSOR_URI}"
+        query = {
+            'limit': 100,
+            'offset': 0,
+            'sortDirection': 'ASC',
+            'sortingFieldName': 'machineName',
+            'filters': [
+                {'fieldName': "guid", 'operator': 'Equals', 'values': guids},
+                {"fieldName": "status", "operator": "NotEquals", "values": ["Archived"]}
+            ]
+        }
+        data = self.instance.execute_query("POST", sensors_url, query)
+        return data.get('sensors', [])
+
+    def _suspicions(self, alert_id: str, last_update_time: int) -> (dict, dict):
+
+        if len(self.DESC_TRANSLATION) <= 0:
+            url = f"{self.instance.host}/{self.DESCRIPTION_URI}"
+            self.DESC_TRANSLATION = self.instance.execute_query("GET", url)
+
+        process_status, suspicions_process = self._suspicions_process(alert_id, last_update_time)
+        network_status, suspicions_network = self._suspicions_network(alert_id)
```

Comment:
```suggestion
        process_status, suspicions_process = self._suspicions_process(alert_id, last_update_time)
        if not process_status:
            msg = f"Could not get process suspicions for alert {alert_id}: {suspicions_process}"
            logger.warning(f"[{__parser__}]:_suspicions: {msg}", extra={'frontend': str(self.frontend)})
        network_status, suspicions_network = self._suspicions_network(alert_id)
        if not network_status:
            msg = f"Could not get network suspicions for alert {alert_id}: {suspicions_network}"
            logger.warning(f"[{__parser__}]:_suspicions: {msg}", extra={'frontend': str(self.frontend)})
```
(please test this suggestion, syntax errors might exist)